community/contributing/index.md

```diff
@@ -8,22 +8,34 @@ redirect_from: /docs/contributing-to-spinnaker
 
 {% include toc %}
 
-## Filing issues
+Welcome to the Spinnaker community! Please join us as a contributor.
 
-Please file issues through the [Spinnaker GitHub issue tracker](https://github.com/spinnaker/spinnaker/issues).
+## Quick Links
 
-## How to Become a Contributor
+- [Spinnaker GitHub issue tracker](https://github.com/spinnaker/spinnaker/issues): file Spinnaker issues
+- [Development environments](/community/gardening/dev-environment/): set up a local development environment to test your pull requests
```

Comment:
I trust that the dev environment setup is suitable for all uses, but because it's under Gardening Days, it might be a little confusing to list it here. Would it be better to move the env setup doc to somewhere outside of Gardening Days, and link to it from there and from here?

Reply:
@dorbin yes that would make sense but I really didn't want to spend a lot of time on the site organization at this point given that it's about to be blown up. I do like the idea of moving that page out of the Gardening area but it would require some edits and I wasn't sure what to do with the existing page.

Reply:
I can try to work that into this PR if desired

Reply:
It does seem like a lot of work to do right before a whole re-do. Let it stand :)